lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -705,13 +738,14 @@
         "abm",
         "avx",
         "xop",
-        "lwp",
```

Comment:
@chissg do you have an example of a bulldozer chip that doesn't have `lwp`? I thought they all supported it.

Reply:
`lwp` is not presented as a flag on RHEL6 and related OS. Requiring this flag means that bulldozer chips are not recognized on these OS.